tests/acceptance/EventCashbookCest.php

```diff
@@ -115,6 +116,7 @@ private function addIncomeChit(): void
 
         $this->fillChitForm(new ChronosDate(), 'Účastnické poplatky', Operation::INCOME(), 'Přijmy od účastníků', 'Testovací skaut 2', '100');
         $this->I->scrollTo('input[name="send"]');
+        $this->I->wait(2);
```

Comment:
proč kdyz to tedka prochazi?

Reply:
Pořád mě to blbne na localhostu. Projde mi to 1x z pěti pokusů. Nechci to dávat úplně pryč...

Reply:
pokud ti to takle projde vzdy tak ok